Gems/Meshlets/ASV_GPU_and_CPU_Demo/Readme.txt

```diff
@@ -116,14 +120,26 @@ Adding the meshoptimizer library to the Gem
 ===========================================
 The meshoptimizer library is not included as part of the O3DE.
 When adding and compiling this Gem, compile the meshoptimizer 
-library and add it as part of Meshlets.Static project (for example, in 
-Visual Studio you simply add the library file to the project).
+library and add it as part of Meshlets.Static project.
 Once this is done, the Gem should compile and link properly to allow you 
-to run the ASV sample 'Meshlets' (created with the MeshletsExampleComponent)
+to run the ASV sample 'Meshlets' (created with the MeshletsExampleComponent).
 
 The meshoptimizer library and source can be found in the following Github link:
 https://github.com/zeux/meshoptimizer
 
+Here are the specific steps to build and install the library, on Windows:
+git clone https://github.com/zeux/meshoptimizer
+cd meshoptimizer
+mkdir build
+cmake . -B build
+cmake --build build --config Release
+copy build\Release\meshoptimizer.lib [YourO3DEPath]\Gems\Meshlets\External\Lib
+
+Finally, in Visual Studio, right-click the Meshlets.Static project 
+    and click Add > Existing Item..., then select 
+    [YourO3DEPath]\Gems\Meshlets\External\Lib\meshoptimizer.lib
+    (Note that if you run cmake in O3DE again, this setting will be lost and you
+    will have to add meshoptimizer.lib to the Meshlets.Static project again).
```

Comment:
another possibility might be to use CMake's `FetchContent` to clone the repo automatically similar to how it is done in this PR https://github.com/o3de/o3de/pull/11626/files#diff-bf1793aef68dee31db8151fb3f89d21b51c97e43e4e7472c084fa06d9d904507

Reply:
Yes, we'll definitely want to tie this into cmake at some point. The meshlets gem was left in a proof-of-concept state. I'm just trying to make minimal updates to the instructions to help readers follow a long, without getting too deep (I'm already outside my intended scope of work).

Reply:
I think that the clear way would be to connect the meshoptimizer as a 3rdParty library but I never got to it.